src/components/button/templates/component/style.js

```diff
@@ -132,181 +128,195 @@ export let componentStyle = `
             height: 18px;
         }
 
-        .paypal-button .paypal-button-content .logo.logo-paypal {
+        .paypal-button .paypal-button-content .logo.logo-paypal,
+        .paypal-button .paypal-button-content .logo.logo-venmo,
+        .paypal-button .paypal-button-content .logo.logo-credit {
             height: 17px;
         }
 
-        .paypal-button.paypal-dual-true .paypal-button-content .logo.logo-paypal,
-        .paypal-button.paypal-dual-true .paypal-button-content .logo.logo-venmo{
-            height: 13px;
+        .paypal-button.paypal-shape-rect .paypal-button-content {
+            border-radius: 4px;
         }
+    }
 
-        .paypal-button .paypal-button-content .logo.logo-credit {
-            height: 16px;
+    @media only screen and (max-width : 147px) {
+
+        .paypal-button .paypal-button-content.paypal-label-credit .logo.logo-paypal {
+            display: none;
         }
 
-        .paypal-button .paypal-button-content::before {
-            padding: 1px;
-            top: -1px;
-            left: -1px;
-            border-radius: 14px;
+        .paypal-button.paypal-dual-true .paypal-button-content:nth-child(2) {
+            display: none;
         }
 
-        .paypal-button.paypal-shape-rect .paypal-button-content,
-        .paypal-button.paypal-shape-rect .paypal-button-content::before {
-            border-radius: 4px;
+        .paypal-button .paypal-button-tag-content {
+            display: none;
         }
     }
 
+    @media only screen and (min-width : 148px) {
+
+        .paypal-button.paypal-dual-true .paypal-button-content .logo.logo-pp {
+            height: 16px;
+        }
+
+        .paypal-button.paypal-dual-true .paypal-button-content .logo.logo-paypal,
+        .paypal-button.paypal-dual-true .paypal-button-content .logo.logo-venmo,
+        .paypal-button.paypal-dual-true .paypal-button-content .logo.logo-credit {
+            height: 15px;
+        }
+
+        .paypal-button.paypal-dual-true .paypal-button-content {
+            width: 49%;
+        }
+
+        .paypal-button.paypal-dual-true .paypal-button-content:nth-child(1) {
+            margin-right: 2%;
+        }
+    }
 
     /* Medium */
 
     @media only screen and (min-width : 200px) {
+
         body {
             height: 48px;
         }
+
         body.fundingicons {
             height: 75px;
         }
-    }
-
-    @media only screen and (min-width : 200px) and (min-height: 48px) {
 
         .paypal-button {
             max-width: 300px;
         }
 
         .paypal-button .paypal-button-content {
-            height: 30px;
-            max-height: 30px;
-            border-radius: 15px;
-            font-size: 12px;
+            height: 32px;
+            border-radius: 16px;
         }
 
-        .paypal-button .paypal-button-content .logo.logo-pp {
-            height: 25px;
+        .paypal-button .paypal-button-content,
+        .paypal-button .paypal-button-content .paypal-tagline {
```

Comment:
` .paypal-button .paypal-button-tag-content {}`
Class name in `template.js` is paypal-button-tag-content for the tag line